hash.c

```diff
@@ -3943,21 +3981,19 @@ rb_hash_values(VALUE hash)
 
 /*
  *  call-seq:
- *     hsh.has_key?(key)    -> true or false
- *     hsh.include?(key)    -> true or false
- *     hsh.key?(key)        -> true or false
- *     hsh.member?(key)     -> true or false
+ *    hash.has_key?(key) -> true or false
```

Comment:
My understanding is Matz had a preference about which one to use... @mame do you know?

Reply:
Also, if we remove the aliases, are they listed elsewhere?

Reply:
Oh, wow.  This is a serious error.  I think I recall that #include? is preferred (by Matz?).  The examples now use #include?, and there's an alias remark.  Resolving for now.